Data/Source/Scripts/AutoTest_Suite_NPCs.psc

```diff
@@ -74,20 +81,38 @@ endFunction
 ; * *baseId* (Integer): The BaseID to clone and take screenshot
 ; * *espName* (String): The name of the ESP containing this base ID
 function ScreenshotOf(int baseId, string espName)
+
   int formId = baseId + Game.GetModByName(espName) * 16777216
   Form formToSpawn = Game.GetFormFromFile(formId, espName)
   string formName = formToSpawn.GetName()
   Log("[ " + espName + "/" + baseId + " ] - [ Start ] - Take screenshot of FormID 0x" + formId + " (" + formName + ")")
   Game.GetPlayer().MoveTo(ViewPointAnchor)
   ObjectReference newRef = TeleportAnchor.PlaceAtMe(formToSpawn)
-  newRef.RemoveAllItems()
+  ; TODO: Add option for clothes here
+
+  string nonNudeNPC = GetConfig("non_nude")
+  if nonNudeNPC != "true"
+    nonNudeNPC = "false"
+  endIf
+
+  if nonNudeNPC == "false"
+    newRef.RemoveAllItems()
+  endIf
+  
   ; Wait for the 3D model to be loaded
   while (!newRef.Is3DLoaded())
     Utility.wait(0.2)
   endWhile
   Utility.wait(1.0)
+
+  ; ensure PC 1st party view (summoning some NPCs seem to change POV)
+  Game.ForceFirstPerson()

```

Comment:
Good catch, I didn't know it was useful. Do you have an example of such an NPC that changes 1st person view?

Reply:
I have atleast one NPC that does it, I think he is from some obscure LoversLab mod I ported from Oldrim, But I figured if I had the issue, someone else might as well.